### PR TITLE
Update Deutsch locale

### PR DESCRIPTION
**Pull request by TooWaBoo**

### CODE REVIEW

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -2278,11 +2279,11 @@ ZOOMPANEL_ZOOMOUT;Herauszoomen\nTaste: <b>-</b>
...
-!SAMPLEFORMAT_0;Unknown data format
-!SAMPLEFORMAT_1;8-bit unsigned
-!SAMPLEFORMAT_2;16-bit unsigned
-!SAMPLEFORMAT_4;24-bit LogLuv
-!SAMPLEFORMAT_8;32-bit LogLuv
-!SAMPLEFORMAT_16;16-bit floating-point
-!SAMPLEFORMAT_32;24-bit floating-point
-!SAMPLEFORMAT_64;32-bit floating-point
+SAMPLEFORMAT_0;Unbekanntes Datenformat
+SAMPLEFORMAT_1;8 Bit ohne Vorzeichen
```
Comment (Beep6581) [inline rtdata/languages/Deutsch:2283]: @TooWaBoo in English the notation is "x-bit": singular "bit" with a hyphen. When I checked the German version of a page on Wikipedia it used the same notation, so maybe the correct German form here should be e.g.
"16-Bit Gleitkomma"?

**Comment by TooWaBoo** [comment]
In German you use the hyphen when it's one word like "16-Bit-Register" or "400-Euro-Job". If you count the bits it will be written without hyphen "16 Bit" or "16 Meter"

**Comment by Beep6581** [comment]
@TooWaBoo So in English in this case we don't count the bits. I believe it's called an "adjective phrase". What kind of apple? Dark-red apple. What kind of file? 16-bit file.

**Comment by TooWaBoo** [comment]
Checkou this
https://books.google.de/books?id=6x2V6yVpqKoC&pg=PA169&lpg=PA169&dq=16+Bit+oder+16-Bit+adobe&source=bl&ots=Cm0YAbyJU8&sig=nIS4zwyNQy-cb_CXwdFwTtAgG-o&hl=de&sa=X&ved=0ahUKEwi_3OfQ_ZHbAhVQYVAKHdtoByM4ChDoAQgnMAA#v=onepage&q=16%20Bit%20oder%2016-Bit%20adobe&f=false

https://www.computerwissen.de/bildbearbeitung/photoshop-tutorials/artikel/wann-16-bit-farbtiefe-sinnvoll-sind.html

https://www.flyeralarm.com/blog/de/serie-bildbearbeitung-die-farbtiefe-8-vs-16-bit/
https://www.heise.de/foto/artikel/16-Bit-fuer-alle-228538.html

**Comment by Beep6581** [comment]
@TooWaBoo there are plenty of books and official websites with incorrect grammar in English, and I don't speak German, so I don't see the point in showing me some random German websites. I wanted to raise awareness about the `SAMPLEFORMAT_` keys, that's all.

**Comment by TooWaBoo** [comment]
What do you expect I should do? In this context "16 Bit" is correct.

**Comment by Beep6581** [comment]
@TooWaBoo I just hope you will do what's best :) In fact, I know you will.

**Comment by Floessie** [comment]
@Beep6581 I'm not a good speller in German, but I think @TooWaBoo is right here. I regard her/him competent enough to make a decision here: She/he has been doing a good job in the past and will surely do so in the future. :+1: 

Just my 2¢,
Flössie

**Comment by TooWaBoo** [comment]
@Floessie 
"him" is correct. 😀

**Pull request merged**